### PR TITLE
GEOMESA-2188 Include GEOMESA_OPTS and CUSTOM_JAVA_OPTS in scala-conso…

### PR DESCRIPTION
**Pull request by aheyne**
…le command.

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-tools/bin/common-functions.sh [hunk 1]
```diff
@@ -302,6 +302,7 @@ function registerAutocomplete() {
 
 function geomesaScalaConsole() {
   classpath=${1}
+  OPTS=${@}
```
Comment (elahrvivaz) [inline geomesa-tools/bin/common-functions.sh]: will this also pick up the classpath arg?

**Comment by aheyne** [comment]
Testing

File: geomesa-tools/bin/common-functions.sh (hunk 1 above)
Comment (aheyne) [reply geomesa-tools/bin/common-functions.sh]: Yeah, my bad, it's fixed.

Comment (aheyne) [reply geomesa-tools/bin/common-functions.sh]: It's not showing up though...

**Pull request merged**